crates/rspack_plugin_javascript/tests/fixtures/builtins/minify/index.js

```diff
@@ -20,7 +20,7 @@
  Foo Bar
  */
 
-// @license 
```

Comment:
should align to terser, and keep the comments format. Currently,  it will be transformed to a block comment anyway.